apps/src/applab/firebaseCounters.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+/* global Applab */
+
+const Firebase = require("firebase");
```

Comment:
I think I've also seen it not work in some cases, and am not sure exactly where. It's also possible that those cases went away with the move from browserify to webpack.